sched/group/group_argvstr.c

```diff
@@ -60,16 +61,20 @@
 
 size_t group_argvstr(FAR struct tcb_s *tcb, FAR char *args, size_t size)
```

Comment:
Also the name group_argvstr is now obsolete, as the argument vector has been moved to TLS. Not a problem of this patch, but something worth noting that it needs clean-up.

Reply:
Should we move `group/group_argvstr.c` to `task/task_argvstr.c` and rename it to `nxtask_argvstr`?

Reply:
That sounds good but I'm starting to wonder if https://github.com/apache/nuttx/pull/12320/files#diff-4c80631b4969dc86c856de89c0a424189374e6284a10a8529662f9e15e3dccd0 is wrong.

It removes the argument vector from group, which I think is wrong. Now pthreads do not inherit the process argument vector.

Reply:
Forget what I said, renaming to task_argvstr is just fine.